tests/importer/importer.cpp

```diff
@@ -1466,3 +1466,27 @@ TEST(Importer, importCascadingUnitsImports)
     const std::string a = printer->printModel(flatModel);
     EXPECT_EQ(e, a);
 }
+
+void testImporterWithInvalidImportedModels(bool strict)
+{
+    const std::string e = "The attempt to import the model at '" + resourcePath("importer/triangle_units_opposite.cellml") + "' failed: the file is not valid XML.";
```

Comment:
Is there no way to get more information from the parser as why the file is not valid XML? I imagine that this is because `xmlns:xlink="http://www.w3.org/1999/xlink"` is missing from the `import` element? What kind of error are we getting from libxml2, if any?

Reply:
I don't think we want to start trying to interpret LibXml2 error messages.  We expect a valid XML document, if we don't get one users are going to have to fix that themselves.  I really feel that XML errors are not our domain.

Reply:
Agreed about not wanting to interpret libxml2 error messages, but I would still provide them, maybe as some kind of additional information. As it stands, the current error message is likely to be of no use to someone who is not familiar with XML and we don't expect CellML users to be familar with XML, so...?

Reply:
I think this is a different issue.  We have talked about multi-level error messages previously.  My recollection is that we would consider this post version 1.0.0.

Reply:
Agreed that this has nothing to do with this PR _per se_ and if this is something that we are willing to consider post 1.0.0 then I am fine with it.

Reply:
I will say that libsbml also does indeed report libxml2 error messages to the user.

Also, the most common XML error is the user giving the library a file that is not XML at all, so that might be worth calling out:  either say 'this file isn't in the correct format'  or 'this file is empty'.  (I forget if an empty file qualifies as 'xml' in libxml2's world.)